modules/swagger-parser-v2-converter/src/test/java/io/swagger/parser/test/V2ConverterTest.java

```diff
@@ -852,6 +853,29 @@ public void testissue1261() throws Exception {
 
     }
 
+    @Test(description = "OpenAPI v2 converter - verifies the additionalProperties")
+    public void testissue1369() throws Exception {
+        OpenAPI oas = getConvertedOpenAPIFromJsonFile(ISSUE_1369_YAML);
+        assertNotNull(oas);
+        Schema schema = (Schema) oas.getComponents().getSchemas().get("Foo1");
+        assertNotNull(schema);
+        assertNull(schema.getAdditionalProperties());
+
+        schema = (Schema) oas.getComponents().getSchemas().get("Foo2");
+        assertNotNull(schema);
+        assertNotNull(schema.getAdditionalProperties());
+        assertEquals(schema.getAdditionalProperties(), Boolean.TRUE);
+
+        schema = (Schema) oas.getComponents().getSchemas().get("Foo3");
+        assertNotNull(schema);
+        assertNotNull(schema.getAdditionalProperties());
+        assertEquals(schema.getAdditionalProperties(), Boolean.FALSE);
```

Comment:
getAdditionalProperties() value should be null at this step in processing for a false spec input

Reply:
My mistake, this should be Boolean.FALSE per checking results when parsing a v3 spec.
This is good